.github/workflows/cli-tests.yml

```
@@ -25,7 +25,7 @@ jobs:
           - name: linux
             image: ubuntu-20.04 # Focal Fossa
           - name: mac
-            image: macos-12
+            image: macos-14
```

Tests failed for `macos-14` as well (see earlier comment). Trying 13.
```suggestion
            image: macos-13
```